### DISCUSSION TITLE
Positionsnummer Widgets

### DISCUSSION THREAD

**Discussion by domi1893** [Zefau/ioBroker.jarvis#1819] [Q&A] [answered]
Hi,

nutze Jarvis 3.1.0-alpha.35 und bin gerade dabei meine Heizung zu integrieren. Jetzt habe ich einfach Mal alle Widgets reingeladen und wollte dann die Anordnung machen. Funktioniert am PC auch relativ gut, wenn ich allerdings das ganze am Handy aufmache ordnet er die Widgets nach der "responsive Position" Reihenfolge an. Diese Nummer scheint sich auch nicht zu ändern, wenn ich die Widgets verschiebe. Gibt es eine Möglichkeit diese manuell zu ändern?

Danke und Grüße,

Domi

**Comment by mcuiobroker** [comment, marked as answer]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/system-effektprog./v3-systemprogramme/move-widget-ordernumber-in-layout

**Reply by domi1893** [reply]
Thx, hab ich irgendwie nicht gefunden...

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/571
